pack/config/whitelist.txt

```diff
@@ -76,6 +80,7 @@ ffxiv*                       || Final Fantasy XIV Launcher or Configurator
 FirefallClient               || Firefall
 FoxGame-Win32-Shipping       || Blacklight: Retribution
 fr-041_debris                || "Debris" by Farbrausch
+Game                         || Gurumin: A Monstrous Adventure
```

Comment:
It does. I really need to implement path-based detection at some point.
(Or devs need to give better names to their executables :P)